recipe/meta.yaml

```diff
@@ -12,8 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  script: DISABLE_NUMCODECS_AVX2= python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
So you could either break it out as separate scripts or you could have a multiline script. I guess the latter has been somewhat discouraged, but for short things like this it should be fine. Here's an [example]( https://github.com/conda-forge/waf-feedstock/blob/1c0892e2d4e31feba6add875a097a37bcc2401af/recipe/meta.yaml#L15-L18 ) of the latter and [one]( https://github.com/conda-forge/scikit-image-feedstock/tree/5e0dc93cfc8e632a8f72b45fcbde2b97bbc70661/recipe ) of the former.